packages/suite/src/hooks/wallet/useStakeEthForm.ts

```diff
@@ -181,9 +198,11 @@ export const useStakeEthForm = ({ selectedAccount }: UseStakeFormsProps): StakeC
         async (amount: string) => {
             setIsAmountForWithdrawalWarningShown(false);
             setIsAdviceForWithdrawalWarningShown(false);
-            if (!fiatRates || !fiatRates.current) return;
+            if (!currentRate) return;
```

Comment:
This condition is reason for number 3., there is no current rate for testnet coins

Reply:
https://github.com/everstake/trezor-suite/assets/33235762/f1e2ba76-37c7-47d8-a7a0-ce31fa0a5a33